### ISSUE TITLE
Merge/combine label tracks

### ISSUE THREAD

**Issue by eharris** [audacity/audacity#6034]
### Your idea

Provide a way to merge/combine Label tracks within Audacity.

### Problem to be solved

I often use `Analyze` -> `Label Sounds` to detect cut points within an audio track.
However, `Label Sounds` refuses to work if the selection is too long (an error is shown: `Selection must be less than 13h 31m 35s`)
To work around this, I select segments less than 13.5 hours and apply `Label Sounds` to each segment.  Unfortunately this creates new label tracks rather than adding new labels into the the existing label track.
Multiple label tracks might be ok (although annoying), except that `File` -> `Export` -> `Export Multiple` -> `Split files based on: Labels` only seems to use the first label track.
So, I would like an easy way to merge these multiple label tracks into one, without having to export and then re-import them.

### Additional context

My workaround is currently to select all label tracks, export labels, delete all label tracks, and then import labels.  But this is rather tedious and it would be great to just be able to merge the label tracks within Audacity.

I found this topic on the forum, but the last comment doesn't seem to work as it's for a much older version of Audacity:
https://forum.audacityteam.org/t/merging-label-tracks/33178/2

Alternate improvements that would help resolve this issue:
1. Fix `Label Sounds` to allow selections longer than 13.5 hours
2. Improve `Label Sounds` to allow either creating a new Label track or adding the labels to an existing Label track
3. Add some way to cut/copy/paste a group of labels between label tracks as there seemed to be in that older version of Audacity (from the above forum post link).

**Comment by petersampsonaudacity** [comment]
See:   **Allow Label Sounds to work on selections larger than 13.5 hours #6035**

**Comment by petersampsonaudacity** [comment]
@eharris 
Yes, both those old workarounds no longer work in later Audacity versions

But try this

1.  Get some audio with a couple of label tracks
![image](https://github.com/audacity/audacity/assets/50205984/1d912836-8337-4a55-9ce3-2976b3b5b5b3)


2.  Click in on of the label tracks with your cursor
3. File > Export Other > Export labels
4.  Observe:  This now creates a TXT file (no longer XLS as it used to be).
![image](https://github.com/audacity/audacity/assets/50205984/d5f957cd-740c-4ab3-8023-f68bee00f0f0)
5. Observe the TXT file is not time ordered (but that doesn't seem to matter)
6. File > Import

**Comment by petersampsonaudacity** [comment]
1.  The Export labels command only operates if there is a cursor position or selection in at least one label track in the project

2. Export labels appears to export **ALL** labels from ALL tracks regardless of what selection is made in a label track or label tracks.

@LWinterberg @dozzzzer 
Are both of these (especially 2) desired behaviors?  
If not should we be raising a bug (or bugs) for this?

**Comment by eharris** [comment]
@petersampsonaudacity yes, I am already using export/import of labels to work around this issue (as noted in the OP)
